tests/func/parsing/test_interpolated_entry.py

```diff
@@ -315,3 +321,19 @@ def test_cmd_dict(tmp_dir, dvc, bool_config, list_config):
             }
         },
     )
+
+
+def test_interpolate_bool(tmp_dir, dvc):
```

Comment:
We don't need this test. This is already tested:

https://github.com/iterative/dvc/blob/071a5b7ba4a545aa418e880127bc34f60fe0a3b0/tests/unit/test_context.py#L409